openwisp_radius/api/utils.py

```diff
@@ -25,3 +30,27 @@ def is_sms_verification_enabled(org):
         raise APIException(
             _('Could not complete operation ' 'because of an internal misconfiguration')
         )
+
+
+class ThrottledAPIMixin(object):
```

Comment:
Moved these mixins here because they were used in both `views` and `freeradius_views`